{{cookiecutter.project_slug}}/CONTRIBUTING.rst

```diff
@@ -279,24 +279,7 @@ Tips
 
 To run a subset of tests::
 
-{% if cookiecutter.use_pytest == 'y' -%}
-    $ pytest tests.test_{{ cookiecutter.project_slug }}
-{% else %}
-    $ python -m unittest tests.test_{{ cookiecutter.project_slug }}
-{%- endif %}
-
-Deploying
----------
-
-A reminder for the maintainers on how to deploy.
-Make sure all your changes are committed (including an entry in HISTORY.rst).
-Then run::
-
-$ bump2version patch # possible: major / minor / patch
-$ git push
-$ git push --tags
-
-Travis will then deploy to PyPI if tests pass.
+$ pytest tests.test_{{ cookiecutter.project_slug }} -k <test_name_prefix>
 
 Code of Conduct
 ---------------
```

Comment:
we have a seperate code of conduct file now, should we remove that part?

Reply:
Really, where is it?

Reply:
{{cookiecutter.project_slug}}/CODE_OF_CONDUCT.rst

Reply:
Yes, I know and it is pointing to it. Check line 287.

Reply:
oh, right, sorry